syntax = "proto3";

enum StoreResult {
    STORE_SUCCESSFUL = 0;
    STORE_FAILED = 1;
}